database/dtb.ksy

```diff
@@ -0,0 +1,94 @@
+meta:
+  id: dtb
+  title: Flattened Devicetree Format
+  file-extension: dtb
+  license: CC0-1.0
+  endian: be
+doc: |
+  Also referred to as Devicetree Blob (DTB). It is a flat
+  binary encoding of data (primarily devicetree data, although
+  other data is possible as well).
+doc-ref:
+  - https://github.com/devicetree-org/devicetree-specification/releases/download/v0.3/devicetree-specification-v0.3.pdf
```

Comment:
I guess https://github.com/devicetree-org/devicetree-specification/blob/master/source/chapter5-flattened-format.rst makes more sense.

Reply:
I will add this. The PDF has all data in one document, which can be convenient for some.